manifests/params.pp

```diff
@@ -57,13 +57,15 @@
 
   case $facts['os']['name'] {
     'windows': {
-      $binary_group = 'Administrators'
-      $binary_mode = '0755'
+      $data_dir_mode = '0775'
+      $binary_group = undef
+      $binary_mode = '0775'
       $binary_name = 'consul.exe'
-      $binary_owner = 'Administrator'
+      $binary_owner = 'NT AUTHORITY\NETWORK SERVICE'
```

Comment:
It's more correct to run the service as NETWORK SERVICE and not the local Administrator.